### PR TITLE
[NGPDCP-3292] Add dialog mobile examples

### PR DESCRIPTION
**Pull request by pieliedie**
# Merge request description template
image

## Description
- add examples for dialog mobile.
- fix bug in Confirm pickup, dropdown textfield.
**1) Root cause**

**2) Changes**

**3) Impact**

## Check list

- [x] 1. Did you start and verify your changes?
- [x] 2. Did you run build to make sure that your changes can be built successfully?
- [x] 3. No !important flag, inline style in css
- [x] 4. No boilerplate codes (1)
- [x] 5. No duplicate code that exist in common functions?
- [x] 6. All of defined variables should have default value, check null and undefined?
- [x] 7. Use meaningful name for variables, no suffix 1,2,... Describes reference information for easier to understand what's inside. (2)
- [x] 9. Unsubscribed all of subscribers and even listeners when you don't need them.

```
(1)
Boilerplate codes is duplicated multiple times a bunch of the same code. This should be a common function to reuse through your code or you can use generic class / methods or design pattern to avoid the Boilerplate codes.
```

```
(2)
Follow theo coding convention
NO acronym variable name:
WRONG: el, elm
RIGHT: element
Your code can be read as a sentence
```

### CODE REVIEW

**Comment by fpt-duongtx** [comment]
Need spacing between buttons and textarea, @pieliedie !

File: packages/react-compass/src/month-year-picker/wheel.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+      values.push({style, value})
+    }
+    return values
+  }
+
+  return (
+    <>
+      <StyledWheel perspective={perspective} ref={sliderRef}>
+        <div
+          className='wheel__shadow-top'
```
Comment (fpt-duongtx) [inline packages/react-compass/src/month-year-picker/wheel.tsx:96]: Please no BEM style in our class name!

File: packages/react-compass/src/month-year-picker/wheel.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+  loop?: boolean
+  perspective?: 'left' | 'right' | 'center'
+  setValue?: (relative: number, absolute: number) => string | number
+  width: number
+  onChange: (details: TrackDetails) => void
+}) {
+  const perspective = props.perspective || 'center'
+  const wheelSize = 20
+  const slides = props.length
+  const slideDegree = 360 / wheelSize
```
Comment (fpt-duongtx) [inline packages/react-compass/src/month-year-picker/wheel.tsx:17]: Convert magic number into constant

File: packages/react-compass/src/dropdown-textfield/dropdown-textfield.tsx [hunk 3]
```diff
@@ -118,20 +118,24 @@ const DropdownTextfield = React.forwardRef<
...
-  React.useEffect(() => {
-    if (isFirstRun.current) {
-      isFirstRun.current = false
-    } else {
-      if (onDropdownInputChange) {
-        onDropdownInputChange(String(selectedDropdownKey), textfieldValue)
-      }
-    }
-  }, [selectedDropdownKey, textfieldValue])
+  // React.useEffect(() => {
```
Comment (fpt-duongtx) [inline packages/react-compass/src/dropdown-textfield/dropdown-textfield.tsx:128]: Should remove unused code block

**Comment by fpt-duongtx** [comment]
Why do we need keen-slider?

**Review by hoapn-fsoft — APPROVED** [review]

**Review by Tai-Chi-Bui — APPROVED** [review]

**Pull request merged**